Buttplug.Client/ButtplugWSClient.cs

```diff
@@ -107,7 +106,7 @@ public ButtplugWSClient(string aClientName)
         {
             if (_ws != null && (_ws.State == WebSocketState.Connecting || _ws.State == WebSocketState.Open))
             {
-                throw new AccessViolationException("Already connected!");
```

Comment:
AccessViolationException is intended for unsafe or unmanaged code. InvalidOperationException is much more appropriate here. (See: https://docs.microsoft.com/en-us/dotnet/api/system.accessviolationexception?view=netframework-4.7.1&viewFallbackFrom=netstandard-1.4)